msticpy/datamodel/entities/file.py

```diff
@@ -141,19 +148,27 @@ def _create_from_event(self, src_event, role):
         # Sha256Ac (type System.String)
         "Sha256Ac": None,
         "FileHashes": (list, "FileHash"),
+        "TimeGenerated": None,
```

Comment:
If these are common to all entities we might want to think about implementing this a bit more efficiently.
Have a base ENTITY_SCHEMA dict in Entity and define individual _entity_schema as
{
   **(Entity.ENTITY_SCHEMA),
   "prop1": type
   ...
}
WDYT?